### PR TITLE
[SDK-697] Change to latest setup

### PR DESCRIPTION
**Pull request by bitPogo**
## Description

This adapts the latest iteration of the gradle build setup.

## Motivation and Context

The current build setup up is pretty old.

## How is it being implemented?

n/a

## How Has This Been Tested?

n/a

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the changelog accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by wmontwe — CHANGES_REQUESTED** [review]
Small issues

File: .idea/inspectionProfiles/ktlint.xml [hunk 1]
```diff
@@ -1,13 +1,7 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0">
     <option name="myName" value="ktlint" />
-    <inspection_tool class="Destructure" enabled="true" level="INFO" enabled_by_default="true" />
```
[the remark this answers is not in the document]
Comment (bitPogo) [reply .idea/inspectionProfiles/ktlint.xml:4]: I think it is...maybe we should also include into the profile

File: README.adoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 = Data4Life SDK
-:library_version: 1.15.1
+:link-repository: https://github.com/d4l-data4life/hc-sdk-kmp
+:util-sdk-version: 1.15.1
```
[the remark this answers is not in the document]
Comment (bitPogo) [reply README.adoc]: done

File: .idea/copyright/data4life.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
```
[the remark this answers is not in the document]
Comment (bitPogo) [reply .idea/copyright/data4life.xml]: done

**Review by wmontwe — APPROVED** [review]
👍

**Pull request merged**